app/players/player.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    @abstractmethod
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
```

Comment:
but actually - remove this method at all
Every class in Python has implemented `__init__` method. I think that it is not needed here

Reply:
Ok, I did. But there's this line in the README...
> The base class should be the `Player` class, which should have a single `nickname` attribute, where the player's name will be stored.

Reply:
SORRY, I FORGOT TO REMOVE MY COMMENT
I understood that your solution was right... Can you rollback the `__init__` method? 🥲

Reply:
Sure, no problem! :)